chia/wallet/trade_manager.py

```diff
@@ -246,9 +283,15 @@ async def save_trade(self, trade: TradeRecord):
         self.wallet_state_manager.state_changed("offer_added")
 
     async def create_offer_for_ids(
-        self, offer: Dict[Union[int, bytes32], int], fee: uint64 = uint64(0), validate_only: bool = False
+        self,
+        offer: Dict[Union[int, bytes32], int],
+        driver_dict: Optional[Dict[bytes32, PuzzleInfo]] = None,
+        fee: uint64 = uint64(0),
+        validate_only: bool = False,
     ) -> Tuple[bool, Optional[TradeRecord], Optional[str]]:
-        success, created_offer, error = await self._create_offer_for_ids(offer, fee=fee)
+        if driver_dict is None:
+            driver_dict = {}
```

Comment:
```suggestion
```

Its only passed into `_ create_offer_for_ids` here where it gets created if its `None`.